[2024-11-30T05:26:03.561+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-30T05:26:03.594+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt-dag.dbt_snapshot manual__2024-11-30T05:25:37.353140+00:00 [queued]>
[2024-11-30T05:26:03.602+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt-dag.dbt_snapshot manual__2024-11-30T05:25:37.353140+00:00 [queued]>
[2024-11-30T05:26:03.602+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-30T05:26:03.617+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_snapshot> on 2024-11-30 05:25:37.353140+00:00
[2024-11-30T05:26:03.629+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt-dag', 'dbt_snapshot', 'manual__2024-11-30T05:25:37.353140+00:00', '--job-id', '326', '--raw', '--subdir', 'DAGS_FOLDER/dbt-dag.py', '--cfg-path', '/tmp/tmpn41c2lzc']
[2024-11-30T05:26:03.631+0000] {standard_task_runner.py:91} INFO - Job 326: Subtask dbt_snapshot
[2024-11-30T05:26:03.636+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4018) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-30T05:26:03.637+0000] {standard_task_runner.py:63} INFO - Started process 4019 to run task
[2024-11-30T05:26:03.676+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt-dag.dbt_snapshot manual__2024-11-30T05:25:37.353140+00:00 [running]> on host b09a94790db6
[2024-11-30T05:26:03.815+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt-dag' AIRFLOW_CTX_TASK_ID='dbt_snapshot' AIRFLOW_CTX_EXECUTION_DATE='2024-11-30T05:25:37.353140+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-30T05:25:37.353140+00:00'
[2024-11-30T05:26:03.816+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-30T05:26:03.830+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-30T05:26:03.832+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt snapshot --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt']
[2024-11-30T05:26:03.845+0000] {subprocess.py:86} INFO - Output:
[2024-11-30T05:26:05.989+0000] {subprocess.py:93} INFO - [0m05:26:05  Running with dbt=1.8.7
[2024-11-30T05:26:07.109+0000] {subprocess.py:93} INFO - [0m05:26:07  Registered adapter: snowflake=1.8.4
[2024-11-30T05:26:07.825+0000] {subprocess.py:93} INFO - [0m05:26:07  Found 7 models, 1 test, 4 sources, 459 macros
[2024-11-30T05:26:07.828+0000] {subprocess.py:93} INFO - [0m05:26:07
[2024-11-30T05:26:07.829+0000] {subprocess.py:93} INFO - [0m05:26:07  Nothing to do. Try checking your model configs and model specification args
[2024-11-30T05:26:08.642+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-30T05:26:08.650+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-30T05:26:08.810+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt-dag, task_id=dbt_snapshot, run_id=manual__2024-11-30T05:25:37.353140+00:00, execution_date=20241130T052537, start_date=20241130T052603, end_date=20241130T052608
[2024-11-30T05:26:08.883+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-30T05:26:08.905+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-30T05:26:08.934+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
